docs/versioned_docs/version-2.50.0-LTS/widgets/divider.md

```diff
@@ -2,60 +2,71 @@
 id: divider
 title: Divider
 ---
+
 # Divider
 
-**Divider** widget is used to add separator between components. 
+**Divider** component is used to add separator between components.
+
+<div style={{paddingTop:'24px'}}>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+---
 
 ## Component Specific Actions (CSA)
 
 There are currently no CSA (Component-Specific Actions) implemented to regulate or control the component.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
+
+---
```

Comment:
Hey @AryanK1511, divider was only expected before the style section, please remove the extra dividers wherever you have added.

Reply:
Oh maybe my linter added that! Lemme fix it